index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Quiz 1</title>
+    <link rel="stylesheet" type="text/css" href="main.css">
+</head>
+<body>
+        <article class="cards1">
+        <header>
+            <h1>German Chocolate Cake</h1>
+        </header>
+        <section>
+        <element>
+<img class="byitself" src="http://www.keeponcaking.com/wp-content/uploads/2010/10/finished-cake.jpg" width="400" height="300">
+        </element>
+        <element class="pad">
+            Dark chocolate cake covered in rich German Chocolate icing, topped with strawberries.
+        </element>
+        <element class="special">
+            Available
+        </element>
+        </section>
+        <section class="space", "volume">
+            <h4>Specifications</h4>
+            <element>Size: 12" Round<br>Weight: 24 Ounces</element>
+            <footer>Specifications valid thru Dec. 31, 2016</footer>
+        </section>
+        <section>
+            <h3>Pricing</h3>
+            <p class="price">1 cake - $24.99<br>2 cakes - $45.99<br>3 cakes - $65.99</p>
```

Comment:
Better, since you use a `<p>` tag instead of your custom element tag, but make three `<p>`'s instead; one for each cake quantity